src/portal/lib/src/service/interface.ts

```diff
@@ -290,25 +290,35 @@ export enum VulnerabilitySeverity {
 
 export interface VulnerabilityBase {
   id: string;
-  severity: VulnerabilitySeverity;
+  severity: string;
   package: string;
   version: string;
 }
 
 export interface VulnerabilityItem extends VulnerabilityBase {
-  link: string;
-  fixedVersion: string;
+  links: string[];
+  fix_version: string;
   layer?: string;
   description: string;
 }
 
 export interface VulnerabilitySummary {
-  image_digest?: string;
-  scan_status: string;
-  job_id?: number;
-  severity: VulnerabilitySeverity;
-  components: VulnerabilityComponents;
-  update_time: Date; // Use as complete timestamp
+    report_id?: string;
+    mime_type?: string;
+    scan_status?: string;
+    severity?: string;
+    duration?: number;
+    summary?: SeveritySummary;
+    start_time?: Date;
+    end_time?: Date;
+}
+export interface SeveritySummary {
+  total: number;
+  summary: {[key: string]: number};
+}
+
+export interface ScanOverview {
+  "application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0"?: VulnerabilitySummary;
```

Comment:
"application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0" is this a key of the object? Quite strange.

Reply:
yes, it is special